node/grpc/utils.go

```diff
@@ -40,15 +43,15 @@ func GetBlobMessages(in *pb.StoreChunksRequest) ([]*core.BlobMessage, error) {
 		}
 
 		bundles := make(map[core.QuorumID]core.Bundle, len(blob.GetBundles()))
-		for i, chunks := range blob.GetBundles() {
-			quorumID := blob.GetHeader().GetQuorumHeaders()[i].QuorumId
+		for j, chunks := range blob.GetBundles() {
+			quorumID := blob.GetHeader().GetQuorumHeaders()[j].QuorumId
 			bundles[uint8(quorumID)] = make([]*encoding.Frame, len(chunks.GetChunks()))
```

Comment:
casting from uint32 to uint8 is a bit dangerous, I think 256 can overwrite 0. I feel I am doing a bit too much on the audit side now

Reply:
The protobuf doesn't have uint8, that's why it's uint32 from the gRPC request, and we will cast it to uint8 inside golang world

Reply:
https://github.com/Layr-Labs/eigenda/pull/378/files#r1535903558